bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/QualifierFilterAdapter.java

```diff
@@ -64,25 +63,39 @@ public Filter adapt(FilterAdapterContext context, QualifierFilter filter) throws
   private Filter adaptBinaryComparator(
```

Comment:
Would you mind adding a bit of doc here for the future?

Something along the lines of:
Handles conversion of HBase's binary comparators to Bigtable's filters. Inequalities are implemented as ColumnRange filters, while EQUALS is implemented as a literal column_qualifier_regex_filter